charts/janssen/values.yaml

```diff
@@ -258,6 +258,30 @@ config:
     cnAwsSecretsReplicaRegions: []
     # [aws_secret_manager_envs] END
     # [aws_envs] END
+    # [vault_envs] Envs related to Hashicorp vault 
+    # base URL of Vault (default to `http://localhost:8200`).
+    cnSecretVaultAddr: http://localhost:8200
```

Comment:
@misba7 as this is part of `config.configmap`, the `cnSecretVault*` keys need to be defined in sub-chart config as well.